pyaedt/generic/compliance.py

```diff
@@ -109,7 +110,7 @@ def add_aedt_report(self, name, report_type, config_file, design_name, traces, s
         name : str
             Name of the report.
         report_type : str
-            Report type. It can be "frequency", "eye diagram", "statistical eye", "contour eye diagram", "transient".
+            Report type. It can be "frequency", "eye diagram", "statistical eye", "contour eye diagram", "time".
```

Comment:
```suggestion
            Report type. Options are "contour eye diagram", "eye diagram", "frequency", "statistical eye", and "time".
```
Should be an alphabetized list.